ui/app/components/job-page/parameterized-child.js

```diff
@@ -26,4 +27,8 @@ export default class ParameterizedChild extends PeriodicChildJobPage {
   get nodes() {
     return this.store.peekAll('node');
   }
+
+  get shouldDisplayClientInformation() {
+    return this.can.can('read client') && this.job.hasClientStatus;
+  }
```

Comment:
We should move the shouldDisplayClientInformation into AbstractJob and then we'll have access to this property.